### ISSUE TITLE
Image buttons should have complete alt text 

### ISSUE THREAD

**Issue by sergiobaptista** [TBD54566975/ssi-sdk#96]
https://github.com/TBD54566975/ssi-sdk/blob/main/README.md

In all cases, the alt includes only the text in the gray sections. Make these

godoc ssi-sdk

go version 1.17.6

go report A+

license Apache 2

issue 21 open

ssi-sdk-ci passing

**Comment by angiejones** [comment]
for issues, the number open is a variable so can't hardcode that into the alt text. also for CI, don't want to hard code the status
